client/src/pages/Mates/MatesPage.tsx

```diff
@@ -1,12 +1,325 @@
+  syncs: SyncType[];
+  editingSync: SyncType;
+};
+
+enum SyncModalEnum {
+  TITLE = "Title",
+  DATE = "Date",
+  DETAILS = "Details",
+  SYNC_STATUS = "SyncStatus",
+}
+
+/* Hardcoded data below */
+let id = 0;
+const mateName = "Clare Carlisle";
+const initialSyncs: SyncType[] = [
+  {
+    id: id++,
+    ts: new Date("2021-09-01"),
+    title: "Dinner at Chipotle",
+    details: "Got a chicken bowl with guacs. Talked about K.",
+    syncStatus: SyncStatusEnum.COMPLETED,
+  },
+  {
+    id: id++,
+    ts: new Date("2021-04-09"),
+    title: "Coffee at Starbucks",
+    details:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    syncStatus: SyncStatusEnum.COMPLETED,
+  },
+  {
+    id: id++,
+    ts: new Date("2021-06-19"),
+    title: "Snoozed One",
+    details:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    syncStatus: SyncStatusEnum.SNOOZED,
+  },
+  {
+    id: id++,
+    ts: new Date("2021-07-15"),
+    title: "Busy with Work",
+    details: "",
+    syncStatus: SyncStatusEnum.DECLINED,
+  },
+  {
+    id: id++,
+    ts: new Date("2021-07-21"),
+    title: "Hiking at Stone Mountain",
+    details:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    syncStatus: SyncStatusEnum.COMPLETED,
+  },
+  {
+    id: id++,
+    ts: new Date("2021-08-31"),
+    title: "Flaky?",
+    details:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    syncStatus: SyncStatusEnum.DECLINED,
+  },
+];
+
+const newSync = {
+  id: id++,
+  ts: new Date(),
+  title: "New Title",
+  details: "Details here...",
+  syncStatus: SyncStatusEnum.DECLINED,
+};
+/* Hardcoded data above */
+
+const { confirm } = Modal;
 const { Title } = Typography;
 
-const MatesPage = (): JSX.Element => (
-  <>
-    <Title>Mates!</Title>
-  </>
-);
+/**
+ * MatePage
+ * @returns
+ */
+const MatesPage = (): JSX.Element => {
+  const [state, setState] = useState<MatesPageState>({
+    syncs: initialSyncs,
+    isModalVisible: false,
+    editingSync: newSync,
+  });
+  const { syncs, isModalVisible, editingSync } = state;
+
+  /**
+   * Removes specified sync
+   * @param sync
+   */
+  const removeSync = (sync: SyncType) => {
+    confirm({
+      title: `Are you sure you want to delete this sync: ${sync.title}?`,
+      icon: <ExclamationCircleOutlined />,
+      content: "This operation cannot be reversed",
+      okText: "Delete",
+      okType: "danger",
+      cancelText: "No",
+      onOk() {
+        // TODO server
+        setState({
+          ...state,
+          syncs: state.syncs.filter((s) => s.id !== sync.id),
+        });
+      },
+    });
+  };
+
+  /**
+   * Open modal to edit specified sync
+   * @param sync
```

Comment:
if you're gonna go through the effort of populating the @params, might as well go the whole way and add information here.